rented-properties/src/main/java/org/egov/cpt/util/FileStoreUtils.java

```diff
@@ -41,4 +51,31 @@ public String fetchFileStoreUrl(ExcelSearchCriteria searchCriteria) {
 		return responseMap;
 	}
 
+	@SuppressWarnings("unchecked")
+	public String fetchFileStoreId(File file, Property property) {
+		StringBuilder uri = new StringBuilder("http://localhost:8060//filestore/v1/files");
```

Comment:
The url should come from properties file. It might already exist in application.properties file.

Reply:
fetching url from application.properties file